ts/features/payments/details/components/PaymentsMethodDetailsDeleteButton.tsx

```diff
@@ -0,0 +1,77 @@
+import { IOToast, ListItemAction } from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { Alert, Platform } from "react-native";
+import { WalletInfo } from "../../../../../definitions/pagopa/walletv3/WalletInfo";
+import I18n from "../../../../i18n";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { useIODispatch } from "../../../../store/hooks";
+import { paymentsDeleteMethodAction } from "../store/actions";
+
+type PaymentsDetailsDeleteMethodButtonProps = {
+  paymentMethod?: WalletInfo;
+};
+
+const PaymentsMethodDetailsDeleteButton = ({
+  paymentMethod
+}: PaymentsDetailsDeleteMethodButtonProps) => {
+  const navigation = useIONavigation();
+  const dispatch = useIODispatch();
+
+  const deleteWallet = (walletId: string) => {
+    dispatch(
+      paymentsDeleteMethodAction.request({
+        walletId,
+        onSuccess: () => {
+          IOToast.success(I18n.t("wallet.delete.successful"));
+        },
+        onFailure: () => {
+          IOToast.error(I18n.t("wallet.delete.failed"));
+        }
+      })
+    );
+    navigation.goBack();
+  };
+
+  if (paymentMethod === undefined) {
+    return null;
```

Comment:
why not show an error toast here? While we are talking about an edge case, I believe an user would like to be warned about things going south.